infra/src/main/resources/db/migration/prod/V36__combine_published_switch_tables.sql

```diff
@@ -71,7 +71,9 @@ alter table publication.switch
 
 alter table publication.switch_joint
   add constraint publication_switch_joint_switch_fk
-    foreign key (publication_id, switch_id) references publication.switch (publication_id, switch_id);
+    foreign key (publication_id, switch_id) references publication.switch (publication_id, switch_id),
+  alter column location_track_external_id drop not null,
+  alter column track_number_external_id drop not null;
```

Comment:
Tämä käytännössä siksi, että se mahdollistaa Geoviitteen toimivuuden ilman Ratkoa (eli devaajilla lokaalisti ja dev-ympäristössä). Ilman tätä muutosta julkaisujen tekeminen räjähtää, koska jos ratkoa ei ole konffattu päälle tai ylipäätään olemassa, sieltä ei sattuneesta syystä saada haettua external-id:itä